python_modules/libraries/dagster-snowflake/dagster_snowflake/snowflake_io_manager.py

```diff
@@ -34,11 +34,85 @@ def build_snowflake_io_manager(type_handlers: Sequence[DbTypeHandler]) -> IOMana
             from dagster_snowflake import build_snowflake_io_manager
             from dagster_snowflake_pandas import SnowflakePandasTypeHandler
 
+            @asset(
+                key_prefix=["my_schema"]  # will be used as the schema in snowflake
+            )
+            def my_table():  # the name of the asset will be the table name
+                ...
+
             snowflake_io_manager = build_snowflake_io_manager([SnowflakePandasTypeHandler()])
+            @repository
+            def my_repo():
+                return with_resources(
+                    [my_table],
+                    {"io_manager": snowflake_io_manager.configured({
+                        "database": "my_database",
+                        "account" : ...,
+                        "user: ...,
+                        ...
+                    })}
+                )
+
+        The returned Snowflake IOManager can be configured with the following values:
+
+        .. code-block:: YAML
+
+            database: my_database  # name of the database in snowflake
```

Comment:
Do we have a standard way of documenting the config values that a resource accepts? If not, I think it's worth creating one. It feels a little awkward to have it inside the examples section.

Reply:
for resources that aren't in a wrapper function the docs site will automatically document the config ([example](https://docs.dagster.io/_apidocs/libraries/dagster-snowflake#dagster_snowflake.snowflake_resource)) but i don't know if there's a way to force that in the api docs for this io manager since it's wrapped in a function. I'll ask erin and see if she knows